src/ParcelRegistry/Parcel/Parcel.cs

```diff
@@ -18,9 +19,10 @@ public sealed partial class Parcel : AggregateRootEntity, ISnapshotable
             ParcelStatus parcelStatus,
             bool isRemoved,
             IEnumerable<AddressPersistentLocalId> addressPersistentLocalIds,
-            Coordinate? xCoordinate,
-            Coordinate? yCoordinate)
+            ExtendedWkbGeometry extendedWkbGeometry)
         {
+            GuardPolygon(WKBReaderFactory.Create().Read(extendedWkbGeometry));
```

Comment:
I'm not 100% we want to perform this check when migrating? It seems logical we only want valid polygons in our domain, but just checking.

Reply:
I would leave it, @ArneD ?